components/core/src/clp/streaming_archive/MetadataDB.cpp

```diff
@@ -208,6 +212,16 @@ static SQLitePreparedStatement get_files_select_statement(
         );
         clause_exists = true;
     }
+    if (false == file_id.empty()) {
```

Comment:
Minor, but the parameter in the method goes file_path then file_id, so I would expect the same for adding the condition to the SQL statement.